test/info.py

```diff
@@ -140,6 +141,7 @@
     (   0x2202,     'lpc11u35_wizwiki_w7500_eco_if',            None,               'WIZWIKI_W7500ECO'                      ),
     (   0x2203,     'lpc11u35_wizwiki_w7500p_if',               None,               'WIZWIKI_W7500P'                        ),
     (   0x1237,     'sam3u2c_ublox_evk_nina_b1_if',            'sam3u2c_bl',        'U-BLOX-EVK-NINA-B1'                    ),
+    (   0xC005,     'lpc11u35_mtconnect04s_if',                 None,               'MtM-MtConnect04S'                      ),
```

Comment:
The target name should match the platform page. In this case it should be MtConnect04S corresponding to the page https://developer.mbed.org/platforms/MtConnect04S/